rudder-agent/SPECS/rudder-agent.spec

```diff
@@ -768,6 +768,12 @@ rm -f %{_builddir}/file.list.%{name}
 %config(noreplace) /etc/ld.so.conf.d/rudder.conf
 %endif
 
+%config(noreplace) /opt/rudder/etc/uuid.hive
+%config(noreplace) /etc/cron.d/rudder-agent
+%config(noreplace) /etc/init.d/rudder-agent
+%config(noreplace) /etc/default/rudder-agent
+%config(noreplace) /etc/profile.d/rudder-agent.sh
```

Comment:
Then we could have a simple conffile (without noreplace) and rpm would replace the file but keep the old one in a .rpmsave file